scripts/zones/Jugner_Forest/mobs/King_Arthro.lua

```diff
@@ -37,5 +58,20 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    SetServerVariable("[POP]King_Arthro",0);
+   
+    GetMobByID(KING_ARTHRO):setLocalVar("[POP]King_Arthro", 0);
+
+    local RespawnTime = 30;--86700;
+    -- Set temporary respawn of 24 hours + 5 minutes
+    GetMobByID(KNIGHT_CRAB1):setRespawnTime(RespawnTime);
```

Comment:
same here

Reply:
```lua
for knightcrab = mob:getID()-1, mob:getID()-10 do
    GetMobByID(knightcrab ):setRespawnTime(RespawnTime);
end
```

Reply:
Ditto

Reply:
30 sec too short think you meant 300 coz

```
# If one Knight Crab is held for more than 5 minutes, the Knight Crabs will begin to respawn.
```

Reply:
Nope, what you saw there my friend was my temporary 30 sec respawn for testing that I forgot to remove! oof. I noticed it this morning but left it since I'm still testing the other modifications.